Frameworks/Foundation/NSString.mm

```diff
@@ -740,13 +743,11 @@ - (NSString*)description {
  @Status Interoperable
 */
 - (BOOL)isEqualToString:(NSString*)compStr {
-    RETURN_FALSE_IF(!compStr);
-
-    if ([compStr isKindOfClass:[NSString class]]) {
```

Comment:
Yes it would.  Looking at CFStringCompareWithOptions, it can handle both non CF NSStrings as well as CFStrings, so this should be okay.  Still confused about the original intent here :)

---
In reply to: [108258325](https://github.com/Microsoft/WinObjC/pull/2323#discussion_r108258325) [](ancestors = 108258325)